src/controllers/tags.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from "express";
+import { getManager, getRepository } from "typeorm";
+import { Tag } from "../models/tag";
+
+/**
+ * GET /tags
+ * Shows all tags
+ */
+export async function getTags(req: Request, res: Response): Promise<void> {
+  const tags = await getManager().find(Tag);
```

Comment:
```ts
const tags = await Tag.find();
```
Dit is hoe de 'ActiveRecord' manier gaat die ik zelf een stuk simpler vind. #1 heeft enkele voorbeelden.